t/22_ipaddr2.t

```diff
@@ -6,8 +6,10 @@ use Test::Warnings;
 use Test::MockModule;
 use Test::Mock::Time;
 use List::Util qw(any none all);
+use testapi qw(set_var);
```

Comment:
`set_var` was not needed on purpose in this lib UT. Better to say the overall `lib/ipaddr2` was all independent from any variable. All the eventually needed variable was accessed in the test module and forwarded to the lib as function arguments. Please try to use this pattern if possible.